### PR TITLE
Expand area filename area to min size  in file list

### LINKED ISSUE #39591: Folders with very short names are hard to open in Web UI
Seen in 10.9.0 RC1

* at the web UI create a folder namend 'I' 
* click slightly left if the I, the checkbox toggles.
* click slightly right of the I, the details sidebar opens
image

The mousepointer in this position is already too far to the left, although it is only 2 or 3 pixels off.
To the right, one pixel off is already too much. On a thin letter I, the active area seems to be only 3 or 4 pixels wide.

Expected behaviour:
* the active area should be padded left and right, so that very short names still have a sufficiently large target area.
* The mouse should give visual feedback when crossing the border between checkbox and name. Maybe a two pixel inactive gap in between or a different shape?

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
Enhancement: Expand file name area to click

With this change, the file name area to click will expand, this is helpful
with very small file or folder names (e.G single digit).

## Related Issue

- Fixes https://github.com/owncloud/core/issues/39591

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by JammingBen — APPROVED** [review]

File: changelog/unreleased/39592 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Expand file name area to click
+
+With this change, the file name area to click will expand, this is helpful
+with very small file or folder names (e.G single digit).
```
Comment (phil-davis) [inline changelog/unreleased/39592:4]:
```suggestion
with very small file or folder names (e.g. single digit).
```

**Comment by JanAckermann** [comment]
@phil-davis THX, good catch, fixed.

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProvisioning-v2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/55/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareUpdateToRoot-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/79/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiOnSqlite-sqlite-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/178/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareReshareToShares1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/74/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiSharePublicLink1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/71/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareReshareToRoot2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/75/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>cliTrashbin-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/118/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProvisioning-v1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/54/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIFiles-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/129/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIUpload-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/150/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProxySmoke-8-8-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/177/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingExt2-git-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/158/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingIntUsers2-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/145/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProxySmoke-8-1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/170/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIFfSmoke-3-1-firefox-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/161/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIMobileSize-3-2-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/168/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIProxySmoke-3-1-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34022/164/1

**Comment by phil-davis** [comment]
> @phil-davis THX, good catch, fixed.
file-extension-fixed

Looks good now.

**Comment by jnweiger** [comment]
Confirmed fixed in 10.9.0 RC2 -- that was quick. Thank you!

**Pull request merged**